newsroom/push/views.py

```diff
@@ -47,6 +47,10 @@ async def handle_publish_planning(item):
     plan_id = await publisher.publish_planning_item(item, orig.to_dict() if orig else {})
     event_id = await publisher.publish_planning_into_event(item)
 
+    if item.get("related_events"):
+        for events in item["related_events"]:
+            event_id = await publisher.publish_planning_related_events(item, events)
```

Comment:
We won't desire to change the `event_id` used here. The `event_id` stored above is the one we require, as that will include the primary linked Event, which is what is used when sending notifications